template/pyproject.toml.jinja

```diff
@@ -76,22 +76,22 @@ build-backend = "setuptools.build_meta"
 include-package-data = true
 
 [tool.setuptools.dynamic]
-version = {attr = "{{cookiecutter.module_name}}.__init__.__version__"}
+version = {attr = "{{module_name}}.__init__.__version__"}
 
 [tool.setuptools.packages.find]
 where = ["src"]
 
 [tool.setuptools.package-data]
 "*" = ["*.yaml"]
 
-{% if cookiecutter.use_git_tags_for_versioning == 'y' and cookiecutter.plugin_name != "foo-bar" %}
+{% if use_git_tags_for_versioning and plugin_name != "foo-bar" %}
```

Comment:
```suggestion
{% if use_git_tags_for_versioning %}
```

Why is there a special case for plugins named "foo-bar"? Should we remove this?

Reply:
ha! I presume that this is because some tests did weird things. Worth removing and seeing what breaks, if anything (after fixing tests)

Reply:
(ie I agree with you that it technically shouldn't be there. It smells like a hack for our own internal testing convenience.)

Reply:
Related: https://github.com/napari/cookiecutter-napari-plugin/pull/187

We'll have to sync the main branches of cookiecutter-napari-plugin and napari-plugin-template again, there's a few bugfixes made to the cookiecutter now.